sklearn/preprocessing/data.py

```diff
@@ -1946,3 +1957,149 @@ def transform(self, X):
         """
         return _transform_selected(X, self._transform,
                                    self.categorical_features, copy=True)
+
+
+def _boxcox(x, lambda_x=None):
+    if lambda_x is None:
+        x, lambda_x = stats.boxcox(x, lambda_x)
+        return x, lambda_x
+    else:
+        x = stats.boxcox(x, lambda_x)
+        return x
+
+
+def boxcox(X, copy=True):
```

Comment:
Is there great benefit in providing a function version? When lambda is tuned on the data, it would be a bad idea to run this on train and test data together.

Reply:
@jnothman are we thinking about adding this back in?